apps/rush-lib/tsconfig.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "types": [
       "jest"
-    ]
+    ],
+    "skipLibCheck": true
```

Comment:
The dependency @types/jest build fail in this project. Add this flag to avoid building that file